ush/config.sh.RRFS_CONUS_3km_ens

```diff
@@ -124,7 +125,7 @@ if [[ -n $RESERVATION ]] ; then
   PPN_RUN_POST="40"
 fi
 
-WTIME_RUN_FCST="01:45:00"
+WTIME_RUN_FCST="02:00:00"
```

Comment:
This was to give the graphics a little extra time to complete all the appropriate forecast times for the 24 hour forecast. The job runs simultaneously with the forecast job to pick up output as its available.

Reply:
If we are going for 36 hours, I will need this to be longer for the graphics. I can create a new variable to track that, but can bump this up if won't be a problem for y'all.